src/server/rest/v1/service/UserService.ts

```diff
@@ -302,11 +307,14 @@ export default class UserService {
     // Filter
     const filteredRequest = UserValidator.getInstance().validateUserGetByID(req.query);
     UtilsService.assertIdIsProvided(action, filteredRequest.ID, MODULE_NAME, 'handleGetUser', req.user);
+    // Get dynamic auth
+    const authorizationFilter = await AuthorizationService.checkAndGetUserAuthorizations(req.tenant, req.user, { ID: filteredRequest.ID.toString() }, Action.READ);
```

Comment:
No, the method UtilsService.checkAndGetUserAuthorization() should assign the 'projectedFields' to the 'user' right away as AuthorizationService.checkAndGetUserAuthorizations() is already called in UtilsService.checkAndGetUserAuthorization().